drools-wb-services/drools-wb-verifier/drools-wb-verifier-api/src/main/java/org/drools/workbench/services/verifier/api/client/ParameterizedCommand.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.services.verifier.api.client;
+
+/**
+ * A command representing a future activity. This was deliberately created in
+ * addition to the existing GWT Command to allow better re-use of menu
+ * structures when a WorkbenchPart is embedded within Eclipse.
+ */
+public interface ParameterizedCommand<T> {
```

Comment:
Ok. Need to keep this in for now, but removing the comment since it is confusing. This is not here because of Eclipse.